src/pybind/mgr/dashboard/requirements-test.txt

```diff
@@ -3,4 +3,4 @@ pytest-instafail
 pyfakefs==4.5.0
 jsonschema~=4.0
 PyJWT~=2.0
-xmlsec==1.3.13 # Pinning because of https://github.com/xmlsec/python-xmlsec/issues/314
+xmlsec==1.3.14 # Pinning because of https://github.com/xmlsec/python-xmlsec/issues/314
```

Comment:
can you please revert this change. 
if you have some local python issues, just change that in your local branch without pushing it to the remote one